modules/update-custom-certificate-central.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-custom-certificates.adoc
+:_module-type: PROCEDURE
+[id="update-custom-certificate-central_{context}"]
+= Updating the custom certificate for an existing instance
+
+[role="_abstract"]
+If you are using a custom certificate for Central, you can update it by following the instructions in this section.
+
+.Procedure
+. Delete the existing custom certificate's secret.
```

Comment:
Through out the rest of the procedure, please add blank lines between steps.  I suspect this is more convention than technical.

Please end the instructional text with a colon.